autoload/vista/executive/ctags.vim

```diff
@@ -46,7 +46,7 @@ function! s:BuildCmd(file) abort
   let ft = &filetype
 
   " Refer to tagbar
-  let common_opt = '--format=2 --excmd=pattern --fields=nksSaf --extras= --file-scope=yes --sort=no --append=no'
+  let common_opt = '--format=2 --excmd=pattern --fields=nksSaf --file-scope=yes --sort=no --append=no'
```

Comment:
Is this only for C/CPP? If so, I think it's safer to remove `--extras=` for C/CPP only.

Reply:
Yep, it's only for C/CPP. And it seems that `--extras-c=...` and `--extras-c++=...` are not working the way I expected them to, so I'm thinking about either adding another column to `s:language_opt` (may come in handy if other languages can benefit from extra options) in `uctags.vim` or just hardcoding it here. Which way is more suitable?

Reply:
It's ok to change `common_opt` only as we already have `g:vista_ctags_cmd` for users to configure it if the default option does not work well.

Reply:
So the patch is okay?

Reply:
Only remove `--extras` for C/CPP.

```vim
let common_opt = '--format=2 --excmd=pattern --fields=nksSaf --file-scope=yes --sort=no --append=no'
" Do not pass --extras in order to let uctags handle the tags for anonymous entities correctly.
if t:vista.source.filetype() !=# 'c' && t:vista.source.filetype() !=# 'cpp'
  let common_opt .= ' --extras= '
endif
```

Reply:
Done, amended.